TrackPropagation/Geant4e/python/geantRefit_cff.py

```diff
@@ -20,12 +20,12 @@
                  Type = cms.string('CMSIMField'),
                  StepperParam = cms.PSet(
                      MaximumEpsilonStep = cms.untracked.double(0.01), ## in mm
-                     DeltaOneStep = cms.double(0.001), ## in mm
+                     DeltaOneStep = cms.double(1e-4),## in mm
                      MaximumLoopCounts = cms.untracked.double(1000.0),
```

Comment:
@civanch,
when trying to fix the Geant4e refitter configuration in 11.2.x for PR https://github.com/cms-sw/cmssw/pull/32260 
I had to add the following parameters:
```python
    DeltaChordTracker = cms.double(0.001), ## in mm
    DeltaOneStepTracker = cms.double(1e-4),## in mm
    DeltaIntersectionTracker = cms.double(1e-6),## in mm
    RmaxTracker = cms.double(8000),        ## in mm
    ZmaxTracker = cms.double(11000),       ## in mm
    EnergyThTracker = cms.double(0.2),     ## in GeV
```
shouldn't those be added here as well?